pydoc-markdown.yml

```
@@ -18,5 +19,13 @@ renderer:
   type: docusaurus
   docs_base_path: docs/
   relative_output_path: automated
-  relative_sidebar_path: sidebars.json
+  relative_sidebar_path: sidebar.json
   sidebar_top_level_label: "API Reference"
+  markdown:
+    render_module_header_template: |+
+      ---
+      sidebar_label: {relative_module_name}
+      title: {module_name}
+      toc_max_heading_level: 5
```

```suggestion
      toc_max_heading_level: 5
      unlisted: true
```
So we can keep this for dev purpose only? Thoughts?

---

Ah we don't have "Recent blogs" type section so it won't be visible without having a link to the "automated" docs